README.md

```diff
@@ -1,2 +1,34 @@
 # docker-spark-gpg
-Docker Alpine image with Spark, Python, Java and gpg. Used for local dev environment.
+Docker Alpine image with Spark, Python, Java and gpg. Used for loacal dev environment.
+
```

Comment:
Can you add a line to say that the docker container is automatically published to https://hub.docker.com/r/danhill2802/spark-gpg/

Reply:
Updated README